pkg/blueprint/installer_customizations.go

```diff
@@ -1,10 +1,11 @@
 package blueprint
 
 type InstallerCustomization struct {
-	Unattended   bool             `json:"unattended,omitempty" toml:"unattended,omitempty"`
-	SudoNopasswd []string         `json:"sudo-nopasswd,omitempty" toml:"sudo-nopasswd,omitempty"`
-	Kickstart    *Kickstart       `json:"kickstart,omitempty" toml:"kickstart,omitempty"`
-	Modules      *AnacondaModules `json:"modules,omitempty" toml:"modules,omitempty"`
+	Unattended   bool                          `json:"unattended,omitempty" toml:"unattended,omitempty"`
+	SudoNopasswd []string                      `json:"sudo-nopasswd,omitempty" toml:"sudo-nopasswd,omitempty"`
+	Kickstart    *Kickstart                    `json:"kickstart,omitempty" toml:"kickstart,omitempty"`
+	Modules      *AnacondaModules              `json:"modules,omitempty" toml:"modules,omitempty"`
+	Kernel       *InstallerKernelCustomization `json:"kernel,omitempty" toml:"kernel,omitempty"`
```

Comment:
Maybe a short comment here (or in the commit message) why we need a new customization and we can't just use the existing "KernelCustomization" that are used for disk images. I don't know the details but maybe: `// installer needs their own customizations so that we can tweak the installed system via the existing KernelCustomizations` or something (assuming that is true and the reason of course :)

Reply:
I see now that I only referred to this in the `images` counterpart of this PR: https://github.com/osbuild/images/pull/1727#issuecomment-3154941891